modules/mauth-common/src/main/java/com/mdsol/mauth/MAuthRequest.java

```diff
@@ -239,8 +304,12 @@ public MAuthRequest build() {
         }
       }
 
+      if (messagePayload == null && bodyInputStream == null) {
+        messagePayload = new byte[] {};
+      }
```

Comment:
no longer needed. Constructor already performs this

Reply:
This case may occur when using Build(), move this part into Constructor. https://github.com/mdsol/mauth-jvm-clients/pull/132/commits/ac9b72d9407033a365bbc428dfdd2315ead45826

Reply:
IMO the builder should be accessing the private constructor. But let's leave it there as it allows us to remove the code in the public constructor which takes a messagePayload

Reply:
This is the case that payload maybe not specified and no default value set either when using MAuthRequest.Builder.build() 
        MAuthRequest mAuthRequest = MAuthRequest.Builder.get()
		.withHttpMethod(method)
		.withResourcePath(requestPath)
		.withMauthHeaders(retrieveMauthHeaders(request))
		.withQueryParameters(queryParameters)
		.build();